code/style.css

```diff
@@ -1,58 +1,249 @@
+/*HEADER, BODY AND CARDS SECTION*/
+
 body {
   margin: 0;
-  font-family: "Courier New", Courier, monospace;
+  font-family: Girassol;
+  background-image: url(./castleroom.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+
+header {
+  color: whitesmoke;
+  text-align: center;
+  font-family: Girassol;
+  padding: 27px;
+  background-color: rgba(0,0,0,0.55);
+}
+
+header h1 {
+  margin: 0;
+  padding: 15px;
+  font-size: 65px;
+}
+
+header p {
+  font-size: 20px;
+}
+
+.main {
+  padding: 50px 0px 200px 0px;
 }
 
 p {
   margin: 1px;
   font-weight: 300;
-  background-color: #fff;
 }
 
 .cards {
-  margin: 50px auto;
+  margin: 0px auto;
+  padding: 50px 0px;
   width: 90%;
-  height: 600px;
-  background: #ccc;
   display: flex;
   justify-content: space-around;
   align-items: center;
 }
 
+.loader {
+  border: 16px solid #f3f3f3;
+  border-radius: 50%;
+  border-top: 16px solid blue;
+  border-right: 16px solid green;
+  border-bottom: 16px solid red;
+  border-left: 16px solid pink;
+  width: 60px;
+  height: 60px;
+  -webkit-animation: spin 2s linear infinite;
+  animation: spin 2s linear infinite;
+  position: absolute;
+  left: 70px;
+  top: 250px;
```

Comment:
Semantic order of this properties should always be 1. top, 2. right, 3. bottom, 4. left - basically, clockwise